epigraphhub/data/colombia/extract.py

```diff
@@ -49,9 +49,14 @@ def _table_last_update() -> datetime:
             curr = conn.execute(
                 "SELECT MAX(fecha_reporte_web) FROM colombia.positive_cases_covid_d"
             )
+
+            if "nan" in curr:
+                return datetime.strptime("1975-01-01 00:00:00", "%Y-%m-%d %H:%M:%S")
```

Comment:
Why are you replacing missing dates with this one? isn't it better  to treat it as NaT (not a time)?